### DISCUSSION TITLE
Project Languages in Repository

### DISCUSSION THREAD

**Discussion by ferhatkurkcuoglu** [community/community#109491] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I upload the project I coded in Java to the repository, but the language in which the project was written does not appear in the Languages ​​section. Can you help me ?

**Comment by its21yash** [comment]
I can help you troubleshoot this. Here are a few steps you can take:
1. Check file extensions.
2. Verify repository settings.
3. Commit more Java code.
4. Use language-specific directories (e.g., "src/main/java")
5. Check repository documentation.
6. Manually specify language: If all the above steps fail, some repositories allow you to manually specify the primary language of your project. Look for an option to do this in the repository settings or configuration files.
